ledger-service/http-json-ledger-client/src/main/scala/com/daml/http/LedgerClientBase.scala

```diff
@@ -5,7 +5,7 @@ package com.daml.http
 
 import com.daml.grpc.adapter.ExecutionSequencerFactory
 import com.daml.ledger.client.configuration.LedgerClientConfiguration
-import com.daml.util.ExceptionOps._
+import com.daml.scalautil.ExceptionOps._
```

Comment:
I think `scalautil` is a bit weird, not my best idea yet but I think `scala.util` sounds better.

Reply:
ups, wrong account. I'm here usually only using @realvictorprm ;)

Reply:
While I agree, I just moved it to an existing package/module which I thought was most appropriate for this to live at i.e `libs-scala\scala-utils`.

Reply:
Oh I see... Geez, nothing we can do about then :(

Thanks anyways :)